unison-share-api/src/Unison/Server/Orphans.hs

```diff
@@ -329,10 +329,21 @@ instance ToCapture (Capture "namespace" Path.Path) where
       "E.g. base.List"
 
 instance ToJSON Path.Path where
-  toJSON p = Aeson.String (tShow p)
```

Comment:
Using Show in a ToJSON kinda makes me nervous, so I found the explicit combinator it was calling.